### PR TITLE
grails 6.2.1 release announcement

### PR DESCRIPTION
**Pull request by jamesfredley**
Adjust `date: October 3, 2024` in `2024-10-03-introducing-grails-6-2-1.md` to generate the HTML before October 3, 2024 during testing

### CODE REVIEW

**Comment by sbglasius** [comment]
@jamesfredley Awesome! I like the level of details.

File: posts/2024-10-03-introducing-grails-6-2-1.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    ````shell
+    grails --version
+    ````
+
+## Upgrading Your Existing Applications to Grails 6.2.1
+
+If you already have a Grails application and want to upgrade to the latest version, follow these steps:
+
+1. Open the project in your favorite IDE (preferably JetBrain's IntelliJ IDEA).
```
Comment (osscontributor) [inline posts/2024-10-03-introducing-grails-6-2-1.md:89]: Just a small thing... Re: "JetBrain's IntelliJ IDEA", the company name is "JetBrains" so I think it probably makes sense to use "JetBrains' IntelliJ IDEA" or "JetBrains IntelliJ IDEA" but not "JetBrain's IntelliJ IDEA.

Comment (jamesfredley) [reply posts/2024-10-03-introducing-grails-6-2-1.md]: This has been updated and also fixed on the release it was copied from.

File: posts/2024-10-03-introducing-grails-6-2-1.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    * Grails Wrapper is now an optional feature from Grails Forge via the Grails distribution or [https://start.grails.org/](https://start.grails.org/])
+      * you can now run **./grailsw update-wrapper** to get the latest grails wrapper version based on the Grails version
+
+## Bug Fixes and Improvements
+
+Grails 6.2.1 also addresses several bug fixes and improvements, including:
+
+* Fixes a lack of a “controller” attribute in GSP tags with a “resource” attribute
+* Fixes a bug in LinkGenerator when domain classes have a static id mapping to another property, that is not “id”
+* Fixes a problem finding the main class when running CLI commands
```
Comment (jdaugherty) [inline posts/2024-10-03-introducing-grails-6-2-1.md:36]: It also: "Fixes an edge cases that causes a cast class exception when the same variable name is used as arguments to multiple actions"

Comment (jdaugherty) [reply posts/2024-10-03-introducing-grails-6-2-1.md:36]: https://github.com/grails/grails-core/issues/13486

Comment (jamesfredley) [reply posts/2024-10-03-introducing-grails-6-2-1.md:36]: Updated

**Pull request merged**